edx_proctoring/static/proctoring/js/views/proctored_exam_attempt_view.js

```diff
@@ -171,16 +173,22 @@ edx = edx || {};
                     endPage = dataJson.pagination_info.total_pages;
                 }
 
-                _.each(
-                    dataJson.proctored_exam_attempts,
-                    function(proctoredExamAttempt) {
-                        var isProctored = proctoredExamAttempt.proctored_exam.is_proctored;
-                        var isPractice = proctoredExamAttempt.proctored_exam.is_practice_exam;
-                        var proctoredText = isPractice ? gettext('Practice') : gettext('Proctored');
+                dataJson.proctored_exam_attempts.forEach(function(proctoredExamAttempt, i) {
+                    var isProctored = proctoredExamAttempt.proctored_exam.is_proctored;
+                    var isPractice = proctoredExamAttempt.proctored_exam.is_practice_exam;
+                    var proctoredText = isPractice ? gettext('Practice') : gettext('Proctored');
+                    // eslint-disable-next-line no-param-reassign
+                    proctoredExamAttempt.exam_attempt_type = !isProctored ? gettext('Timed') : proctoredText;
+
+                    // current CSS selectors do not allow the selection of nth-child for a class selector,
+                    // so we are determining a class used for CSS styling here
+                    // eslint-disable-next-line no-param-reassign
+                    proctoredExamAttempt.row_class = 'odd';
+                    if ((i + 1) % 2 === 0) {
```

Comment:
I had to add the plus one because CSS nth-child/nth-of-type selectors start indexing from 1. This code replicates what an nth-of-type selector would do, if that selector could account for class (see here: https://betterprogramming.pub/css-select-nth-element-with-class-a313d080e2bf)